config/routes.rb

```diff
@@ -130,7 +130,9 @@
 
     scope :facilities, module: 'facilities' do
       resources :va, only: %i[index show], defaults: { format: :json }
+      resources :ccp, only: %i[show], defaults: { format: :json }
       get 'suggested', to: 'va#suggested'
```

Comment:
shouldn't this be nested under resources?

Reply:
regarding routing, the idea here was that "facilities" refers to the scope of the facility locator and then "va" would be everything pretaining to the va facilities and "ccp" at the same level would contain the stuff for community care providers.